changelogs/fragments/3798-fix-lxd-connection-option-vars-support.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - lxd - fix the lxd connection plugin to have all inventory option vars work (https://github.com/ansible-collections/community.general/pull/3798).
```

```suggestion
  - lxd connection plugin - fix the lxd connection plugin to have all inventory option vars work (https://github.com/ansible-collections/community.general/pull/3798).
```
I think it's a good idea to be a bit precise here. Which didn't work before, and which do work now?

---

Agreed! Updated the changelog fragment, let me know what you think? See 48750cad8866fa9fc0bb8560ee1f07d8d6f99bee.